scripts/prep_local_devel_env.sh

```diff
@@ -130,37 +136,38 @@ fi
 echo "TLS Cert: tls.crt"
 echo -e "Wrote \n${TLS_KEY_DATA}\n to: ${TLS_KEY}\n"
 # Kill any running broker pods
-oc scale deploymentconfig asb --replicas 0 -n ${ASB_PROJECT}
+deployments scale asb --replicas 0 -n ${ASB_PROJECT}
```

Comment:
Should it not be `kubectl deployments ...` ?

Reply:
Now that I've tracked it down, I see that this is a function call that just so happens to look like the only thing that is missing is the `kubectl` in front. I think if you were to put those functions in `scripts/lib/cluster.sh`, conditionally defined for OpenShift vs Kubernetes, then `asb::cluster::deployments` would clearly read as a library function call.